cmd/login/login.go

```diff
@@ -43,6 +44,12 @@ Verification code: %s
 Press Enter to continue...
 `
 
+const emailNotVerifiedErrorMessage = `
+Email verification is required to log in.
+Please visit the client to receive a verification email:
```

Comment:
I think we should use a different word than "client" here to refer to the web app :smile: 

Maybe "Please visit the website to receive a verification email", or "Please visit the web app to receive a verification email"?

Reply:
I will fix, I might as well ask Lauren if she has any other suggestions 😄

Reply:
Yes. What about:
"Please visit https://www.numerous.com/verify to verify your email before proceeding."
And make it a clickable link

Reply:
@Lasse-numerous  I can make it "https://www.numerous.com/app/verify" if you like that better? But without it has to be part of the www project, so that will take to long, compared to the benefit.

I can't make the link clickable but default (afaik), that depends on what terminal or emulator the users is using.
Am I right @jfeodor?

Reply:
Thats totally fine to add the app in the link

Reply:
The app links seem to be clickable returned by the CLI after fi deployments are clickable for me

Reply:
The terminal emulators make links clickable, if they support it :)

Reply:
I suppose it could be something with the formatting that is not being handled the same way in the terminal emulator that Lasse is using.

Reply:
@laurenalexander-em  suggest
Please log in to the [website](https://www.numerous.com/app/verify) to receive a verification email. Alternatively, you can copy and paste this link into your browser:
https://www.numerous.com/app/verify

Reply:
Well, we cannot make a link like that as far as I know. And I don't think we need to tell a user using a CLI to copy and paste a URL :smile:

Reply:
Should be changed